test/e2e/cache/create_enable_add_with_private_registry_delete.go

```diff
@@ -0,0 +1,283 @@
+package cache
+
+import (
+	"context"
+	"crypto/rand"
+	"encoding/base64"
+	"fmt"
+	"time"
+
+	gardencorev1beta1 "github.com/gardener/gardener/pkg/apis/core/v1beta1"
+	"github.com/gardener/gardener/test/framework"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"golang.org/x/crypto/bcrypt"
+	autoscalingv1 "k8s.io/api/autoscaling/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/utils/ptr"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/gardener/gardener-extension-registry-cache/pkg/apis/registry/v1alpha3"
+	"github.com/gardener/gardener-extension-registry-cache/test/common"
+	"github.com/gardener/gardener-extension-registry-cache/test/e2e"
+)
+
+const upstreamConfigYaml = `version: 0.1
+log:
+  fields:
+    service: registry
+storage:
+  inmemory:
```

Comment:
From: https://distribution.github.io/distribution/#list-of-configuration-options:

```suggestion
  inmemory: # This driver takes no parameters
```

as it was confusing for me why there is only key without value